packages/office-ui-fabric-react/src/components/Icon/Icon.base.tsx

```diff
@@ -48,13 +48,13 @@ export class IconBase extends React.Component<IIconProps, IIconState> {
     const ImageType = (imageLoadError && imageErrorAs) || Image;
 
     // tslint:disable-next-line:deprecation
-    const ariaLabel = this.props['aria-label'] || this.props.ariaLabel;
-    const containerProps = ariaLabel
+    const hasName = this.props['aria-label'] || this.props.ariaLabel || this.props['aria-labelledby'] || imageProps['aria-labelledby'];
```

Comment:
@ecraig12345 since `aria-label` only will work/be allowed on `<RootType>` if it also has a `role="img"` (since otherwise it's a generic element, and does not support naming), how do you feel about having the same `isImage` check for it in containerProps?

i.e.:
```
role: isImage ? undefined : 'img',
'aria-label': isImage ? undefined : this.props.ariaLabel
```